qemu/tests/qemu_guest_agent.py

```diff
@@ -2140,6 +2142,95 @@ def result_check(flag, write_timeout, session):
                               "stamp in guest file." % write_timeout)
         result_check("thaw", write_timeout, session)
 
+    @error_context.context_aware
+    def gagent_check_fstrim(self, test, params, env):
+        """
+        Execute "guest-fstrim" command via guest agent.
+
+        Steps:
+        1) boot up guest with scsi backend device
+        2) init and format the data disk
+        3) get the original blocks of data disk
+        4) create fragment in data disk
+        5) check the used blocks
+        6) execute fstrim cmd via guest agent
+        7) check if the used blocks is decreased
+
+        :param test: kvm test object
+        :param params: Dictionary with the test parameters
+        :param env: Dictionary with test environment.
+        """
+        def get_blocks():
+            """
+            Get the used blocks of data disk.
+            :return: the used blocks
+            """
+            blocks = process.system_output("stat -t %s" % stg_image_filename)
+            return blocks.strip().split()[2]
+
+        session = self._get_session(params, None)
+        self._open_session_list.append(session)
+
+        error_context.context("Format data disk.", logging.info)
+        image_size_disk = params["image_size_stg"]
+        disk_index = utils_misc.wait_for(
+            lambda: utils_disk.get_windows_disks_index(session,
+                                                       image_size_disk), 120)
+        if not disk_index:
+            test.error("Didn't get windows disk index.")
+        logging.info("Clear readonly of disk and online it in windows guest.")
+        if not utils_disk.update_windows_disk_attributes(session, disk_index):
+            test.error("Failed to update windows disk attributes.")
+        mnt_point = utils_disk.configure_empty_disk(
+            session, disk_index[0], image_size_disk, "windows",
+            labeltype="msdos")
+
+        error_context.context("Check the original blocks of data disk.",
+                              logging.info)
+        stg_image_params = params.object_params("stg")
+        stg_image_filename = storage.get_image_filename(
```

Comment:
For those params: `image_params`, `image_filename`, `image_size`,`disk_index`. 
`**_stg` or `stg_**` are all ok for me, but suggest to keep consistent.

Reply:
their name is unified. Thanks.